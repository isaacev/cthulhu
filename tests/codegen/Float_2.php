<?php

namespace Float_2 {
  function main() {
    2.0 + 2.1;
  }
}

namespace {
  \Float_2\main(null);
}
