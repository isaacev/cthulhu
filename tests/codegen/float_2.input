#[entry]
fn main() -> () {
  2.0 + 2.1;
}
