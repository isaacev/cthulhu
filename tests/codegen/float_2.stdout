<?php

namespace float_2 {
  // #[entry]
  function main() {
    2.0 + 2.1;
  }
}

namespace {
  \float_2\main();
}
